jwt/algorithms.py

```diff
@@ -86,6 +86,8 @@ class HMACAlgorithm(Algorithm):
     def __init__(self, hash_alg):
         self.hash_alg = hash_alg
 
+    SHA256, SHA384, SHA512 = hashlib.sha256, hashlib.sha384, hashlib.sha512
```

Comment:
I think I'd prefer to change the style of these everywhere to separate lines and before the `__init__` method. I think it's easier to parse.

```python
class HMACAlgorithm(Algorithm):
    """
    Performs signing and verification operations using HMAC
    and the specified hash function.
    """
    SHA256 = hashlib.sha256
    SHA384 = hashlib.sha384
    SHA512 = hashlib.sha512

    def __init__(self, hash_alg):
        self.hash_alg = hash_alg
```